api/api.go

```diff
@@ -83,17 +98,17 @@ func ListStateStats(w http.ResponseWriter, r *http.Request, d *db.Database) {
 // GetState provides information on a State
 func GetState(w http.ResponseWriter, r *http.Request, d *db.Database) {
 	w.Header().Set("Access-Control-Allow-Origin", "*")
-	st := util.TrimBasePath(r, "api/state/")
+	lineage := util.TrimBasePath(r, "api/state/")
 	versionID := r.URL.Query().Get("versionid")
-	var err error
-	if versionID == "" {
-		versionID, err = d.DefaultVersion(st)
-		if err != nil {
-			JSONError(w, "Failed to retrieve default version", err)
-			return
-		}
-	}
-	state := d.GetState(st, versionID)
+	// var err error
+	// if versionID == "" {
+	// 	versionID, err = d.DefaultVersion(st)
+	// 	if err != nil {
+	// 		JSONError(w, "Failed to retrieve default version", err)
+	// 		return
+	// 	}
+	// }
```

Comment:
Is this code obsolete now?

Reply:
Yes it took the path in argument.
However I not removed it because I didn't know what this piece of code was for

Reply:
From what I can tell, this retrieves the default version when it is not passed as an argument.

Reply:
Should we keep it? 
I don't really see cases where the GetState api might not have a versionID and, moreover, we no longer pass the path there, so we would have to retrieve it from the lineage only and I don't know if this is possible (in the possibility that there can be different paths for the same lineage)